games-strategy/liquidwar/liquidwar-5.6.5.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop toolchain-funcs
+
+DESCRIPTION="Unique multiplayer wargame"
+HOMEPAGE="http://www.ufoot.org/liquidwar/"
+SRC_URI="https://ufoot.org/download/liquidwar/v5/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+RESTRICT="test"
+
+RDEPEND=">=media-libs/allegro-4.2:0[X]"
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-exec-stack.patch
+	"${FILESDIR}"/${P}-gcc10.patch
+)
+
+src_prepare() {
+	default
+
+	sed -i \
+		-e 's:/games::' \
+		-e '/^MANDIR/ s:=.*:= $(mandir)/man6:' \
+		-e "/^PIXDIR/ s:=.*:= ${EPREFIX}/usr/share/pixmaps:" \
+		-e "/^DESKTOPDIR/ s:=.*:= ${EPREFIX}/usr/share/applications/:" \
+		-e '/^INFODIR/ s/=.*/= $(infodir)/' \
+		-e '/^GAMEDIR/ s/exec_prefix/bindir/' \
+		-e '/install/s:-s ::' \
+		-e 's:$(DOCDIR)/txt:$(DOCDIR):g' \
+		-e 's:$(GMAKE):$(MAKE):' \
+		-e "/^DOCDIR/ s:=.*:= ${EPREFIX}/usr/share/doc/\$(PF):" Makefile.in \
+		|| die 'sed Makefile.in failed'
+	sed -i \
+		-e '/^GAMEDIR/ s/$(exec_prefix)/@bindir@/' \
+		-e 's:/games::' src/Makefile.in \
+		|| die "sed src/Makefile.in failed"
+}
+
+src_configure() {
+	tc-export CC
+	econf \
+		--disable-doc-ps \
+		--disable-doc-pdf \
+		$(use_enable x86 asm)
+}
+
+src_compile() {
+	# skip build_doc target wrt bug 460344
+	emake build_bin build_data
+}
+
+src_install() {
+	emake DESTDIR="${D}" install_nolink
+	einstalldocs
+	rm -f "${ED}"/usr/share/doc/${PF}/COPYING
```

Comment:
`|| die`, please.
and no need for `-f`.
And if possible to remove the file in src_prepare, please do so.

Reply:
Done. Also simplified src_prepare